src/ntpd.c

```diff
@@ -936,6 +942,11 @@ static int ntpd_read (void)
 		return (-1);
 	}
 
+	if (ntohs(ik->status) & STA_NANO) {
```

Comment:
I can, however I think this has a little sense. In general, ```ntpd``` and ```collectd``` may run on different distributed systems. One may monitor ```ntpd``` running on Linux, using his ```collectd``` running on FreeBSD host.